default/data/ui/nav/default.xml

```diff
@@ -1,17 +1,17 @@
 <nav search_view="search" color="#005792">
-<collection label="Scirius Security Platform">
```

Comment:
Well, it will also works for Suricata users. The initial Suricata Sensors formulation was nice.

Reply:
Ok, maybe just use the working dashboard as the first menu, so this is not going to be frustrating for selks or the premium version.

Reply:
I modfy the name to fit your need.

Reply:
Suricata Sensors and Scirius Security Platform